src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const sum = days * 40;
```

Comment:
Consider using a constant for the daily rental cost instead of a literal value '40'. This will make the code more readable and maintainable.